packages/jest-mock/src/index.ts

```diff
@@ -874,15 +877,47 @@ export class ModuleMocker {
     }
 
     this._getSlots(metadata.members).forEach(slot => {
+      let slotMock: Mocked<T>;
       const slotMetadata = (metadata.members && metadata.members[slot]) || {};
+
       if (slotMetadata.ref != null) {
         callbacks.push(
           (function (ref) {
             return () => (mock[slot] = refs[ref]);
           })(slotMetadata.ref),
         );
       } else {
-        mock[slot] = this._generateMock(slotMetadata, callbacks, refs);
+        slotMock = this._generateMock(slotMetadata, callbacks, refs);
+
+        //For superclass accessor properties the subclass metadata contains the definitions
+        //for the getter and setter methods, and the superclass refs to them.
+        //The mock implementations are not available until the callbacks have been executed.
+        //Missing getter and setter refs will be resolved as their callbacks have been
+        //stacked before the setting of the accessor definition is stacked.
+        if (
+          slotMetadata.members?.get?.ref !== undefined ||
+          slotMetadata.members?.set?.ref !== undefined
+        ) {
+          callbacks.push(
+            (function (ref) {
+              return () => Object.defineProperty(mock, slot, ref);
+            })(slotMock as PropertyDescriptor),
+          );
+        } else if (
+          (slotMetadata.members?.get || slotMetadata.members?.set) &&
+          slotMetadata.members?.configurable &&
+          slotMetadata.members?.enumerable
+        ) {
+          //In some cases, e.g. third-party APIs, a 'prototype' ancestor to be
+          //mocked has a function property called 'get'.  In this circumstance
+          //the 'prototype' property cannot be redefined and doing so causes an
+          //exception. Checks have been added for the 'configurable' and
+          //'enumberable' properties present on true accessor property
+          //descriptors to prevent the attempt to replace the API.
```

Comment:
```suggestion
          // In some cases, e.g. third-party APIs, a 'prototype' ancestor to be
          // mocked has a function property called 'get'.  In this circumstance
          // the 'prototype' property cannot be redefined and doing so causes an
          // exception. Checks have been added for the 'configurable' and
          // 'enumberable' properties present on true accessor property
          // descriptors to prevent the attempt to replace the API.
```